pkgs/desktops/gnome-3/misc/gpaste/fix-paths.patch

```diff
@@ -1,6 +1,6 @@
 --- a/src/gnome-shell/extension.js
```

Comment:
Build failed like
https://gist.github.com/worldofpeace/c0387bf735a807820dc972da1e0d22be

Reply:
Done. Needed to use unref because autoptr is not available: https://gitlab.gnome.org/GNOME/glib/merge_requests/749